helm/aws-operator/templates/rbac.yaml

```diff
@@ -32,6 +32,7 @@ rules:
     verbs:
       - get
       - list
+      - update
```

Comment:
Permission needs to be set because of updating the spec field instead of the status field